main/src/main/js/start/src/constants/index.js

```diff
@@ -1,4 +1,4 @@
-export const API_URL = "https://micronaut-starter-staging-ucxwqnh6ka-uc.a.run.app";
+export const API_URL = "http://localhost:3100";
```

Comment:
Revert this one

Reply:
Dang Stashes.... Fixed here https://github.com/micronaut-projects/static-website/pull/46/commits/89f691b5fb3a20edab73ca406fe53ef5b1a86fb6